x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_actions_popopver.scss

```
@@ -0,0 +1,9 @@
+.ruleActionsPopover {
+  .euiContextMenuItem:hover {
+    text-decoration: none;
+  }
+}
+
+button[data-test-subj='deleteRuleButton'] {
+  color: $euiColorDanger;
+}
```

Hey @ersin-erdal one suggestion from best-practices standpoint, we usually like to keep testing and style selectors separate so they don't affect one another if they change in the future. Context menu items should accept a custom `className` which you can then use here to update your styles.

Also, when coloring text, you should always use the `[text` variant](https://elastic.github.io/eui/#/theming/colors#text-colors?themeLanguage=sass).

```suggestion
.ruleActionsPopover__deleteButton {
  color: $euiColorDangerText;
}
```

---

Done, thanks for the feedback :)

---

😕 Er, it looks like you're still targeting the `button[data-test-subj='deleteRule']` instead of a class name

---

I missed that part 😬
Fixed now.